backend/app/engine/octant_rewards/__init__.py

```diff
@@ -1,25 +1,47 @@
 from dataclasses import dataclass, field
 from decimal import Decimal
 
+from app.engine.octant_rewards.community_fund.calculator import (
+    CommunityFundCalculator,
+    CommunityFundPercent,
+)
 from app.engine.octant_rewards.locked_ratio import LockedRatio
 from app.engine.octant_rewards.locked_ratio.default import DefaultLockedRatio
 from app.engine.octant_rewards.matched import MatchedRewards
 from app.engine.octant_rewards.matched.default import DefaultMatchedRewards
 from app.engine.octant_rewards.operational_cost import OperationalCost
 from app.engine.octant_rewards.operational_cost.op_cost_percent import OpCostPercent
+from app.engine.octant_rewards.ppf.calculator import PPFCalculatorPercent, PPFCalculator
 from app.engine.octant_rewards.total_and_individual import TotalAndAllIndividualRewards
 from app.engine.octant_rewards.total_and_individual.default import (
     DefaultTotalAndAllIndividualRewards,
 )
 
 
+class OctantRewardsDefaultValues:
+    OPERATIONAL_COST = Decimal("0.25")
+    PPF = Decimal("0.35")
+    COMMUNITY_FUND = Decimal("0.05")
+
+
 @dataclass
 class OctantRewardsSettings:
     locked_ratio: LockedRatio = field(default_factory=DefaultLockedRatio)
     operational_cost: OperationalCost = field(
-        default_factory=lambda: OpCostPercent(Decimal("0.25"))
+        default_factory=lambda: OpCostPercent(
+            OctantRewardsDefaultValues.OPERATIONAL_COST
+        )
     )
     total_and_all_individual_rewards: TotalAndAllIndividualRewards = field(
         default_factory=DefaultTotalAndAllIndividualRewards
     )
     matched_rewards: MatchedRewards = field(default_factory=DefaultMatchedRewards)
+
+    ppf: PPFCalculator = field(
+        default_factory=lambda: PPFCalculatorPercent(OctantRewardsDefaultValues.PPF)
+    )
+    community_fund: CommunityFundCalculator = field(
+        default_factory=lambda: CommunityFundPercent(
+            OctantRewardsDefaultValues.COMMUNITY_FUND
+        )
```

Comment:
Do we have a field for increased staking here? @leoni-q @kgarbacinski  I think we used it in the epoch 2 and might also use it in the future, we can set it to 0 for epoch 3 and going forward for now.

Reply:
we have only `leftover` currently, which includes `increased_staking` and the rest what's left. this story might be good to introduce `increased_staking` concept and decide how we calculate it: https://linear.app/golemfoundation/issue/OCT-1364/metrics-funds-usage-for-e2-should-show-352-eth-for-staking